src/libs/reportUtils.js

```diff
@@ -93,6 +93,16 @@ function isDefaultRoom(report) {
     ], lodashGet(report, ['chatType'], ''));
 }
 
+/**
+ * Whether the provided report is a Domain room
+ * @param {Object} report
+ * @param {String} report.chatType
+ * @returns {Boolean}
+ */
+function isDomainRoom(report) {
+    return lodashGet(report, ['chatType'], '') === CONST.REPORT.CHAT_TYPE.DOMAIN_ALL;
+}
+
```

Comment:
I have to be slightly picky here - can you please add this **below** the `isAnnounceRoom` function to keep things a bit alphabetized? :)

Reply:
Gotta move `isDefaultRoom()` and `isUserCreatedPolicyRoom()` too then.

Reply:
At a minimum we can put `isDomainRoom` and `isDefaultRoom` between `isAnnounceRoom` and `isUserCreatedPolicyRoom` - we don't necessarily need to alphabetize everything at this point, but at least the room functions can look nice

Reply:
Makes sense. That'd be a slippery slope :)

Reply:
Exactly 😆

Reply:
Done.

Reply:
Thanks man